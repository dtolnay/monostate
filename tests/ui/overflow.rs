use monostate::MustBe;

fn main() {
    let _ = MustBe!(1000u8);
}
