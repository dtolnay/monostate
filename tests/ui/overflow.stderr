error: literal out of range for `u8`
 --> tests/ui/overflow.rs:4:21
  |
4 |     let _ = MustBe!(1000u8);
  |                     ^^^^^^
  |
  = note: the literal `1000u8` does not fit into the type `u8` whose range is `0..=255`
  = note: `#[deny(overflowing_literals)]` on by default
